sale_coupon_incompatibility/models/coupon_program.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2021 Tecnativa - David Vidal
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo import _, models
+
+
+class SaleCouponProgram(models.Model):
```

Comment:
```suggestion
class CouponProgram(models.Model):
```

Reply:
I thought that being part of the sales side it was correct to leave the name as it was.

Reply:
Well, technically it can be whatever name, but the convention is to call it as the model name using PascalCase [<sup>*</sup>](https://www.freecodecamp.org/news/snake-case-vs-camel-case-vs-pascal-case-vs-kebab-case-whats-the-difference/)

Reply:
I will keep it in mind, thanks!